Description: The generated code has BangPatterns, newer ghc wants this mentioned explicitly
Author: Joachim Breitner <nomeata@debian.org>
Bug-Debian: http://bugs.debian.org/628302
--- gtk2hs-buildtools-0.12.0.orig/c2hs/c/CLexer.x
+++ gtk2hs-buildtools-0.12.0/c2hs/c/CLexer.x
@@ -63,6 +63,7 @@
 --

 {
+{-# LANGUAGE BangPatterns #-}

 module CLexer (lexC, parseError) where
